### DISCUSSION TITLE
I am not able to search a base branch while creating a pull request

### DISCUSSION THREAD

**Discussion by smattoo** [community/community#129273] [Pull Requests]
### Select Topic Area

Bug

### Body

I am not able to search a base branch while creating a pull request

**Comment by omerson-cruz** [comment]
Same here. I was able to search branches just a few hours ago when I was creating new Pull Request

**Comment by oguzhanmemis-eva** [comment]
same issue pls check

**Comment by jayfaaa** [comment]
alternative :

[repolink]/compare/[Base]...[compare]

example
https://github.com/my_company/my_repo/compare/my_base...my_compare

**Comment by bhchoi-crema** [comment]
The base branch search bar seems missing.
It's quite disturbing to scroll all the way down to find a branch I want when there're a lot of active branches.
Please get it back! Everyone's missing the feature!

**Comment by jeromie** [comment]
Yes, we need the search bar back! Updating the base branch of an existing PR is extremely difficult in a repo with many branches.

**Comment by anuj-teotia** [comment]
Please give the search back.

**Comment by jishnu-scalgo** [comment]
Iam also facing the same issue, please give back search.

**Comment by mehtab-airmeet** [comment]
In the meantime,  you can use tools like `gh` to change the base branch of a pull request

```shell
brew install gh
gh auth login
gh pr edit <PR_NUMBER> --base <NEW_BASE_BRANCH>
```

**Comment by SandeepKhore** [comment]
Iam also facing the same issue

**Comment by atharvcd** [comment]
### :rocket: Simplest solution for the time being
Hey, folks ! It seems like some issue with github. Till the time the issue gets resolved and we get the search branch option back, below solution should work while creating PRs.
Enter the following in the URL to create new pull request - 

`https://github.com/[organization-or-username]/[repository-name]/compare/[base-branch-name]...[feature-branch-name]`

@smattoo I Hope this helps ! :smile:
